Xamarin.Forms.Platform.WPF/Renderers/FrameRenderer.cs

```diff
@@ -93,16 +94,58 @@ void UpdateBorder()
 			}
 			else
 			{
-				Control.UpdateDependencyColor(Border.BorderBrushProperty, new Color(0, 0, 0, 0));
+				Control.UpdateDependencyColor(Border.BorderBrushProperty, Color.Transparent);
 				Control.BorderThickness = new System.Windows.Thickness(0);
 			}
 		}
 
+		protected virtual UpdateShadow(){ 
+			if (Element.HasShadow)
+			{
+				Control.Effect = new DropShadowEffect()
+				{
+					Color = Colors.Gray,
+					Direction = 320,
+					Opacity = 0.5,
+					BlurRadius = 6,
+					ShadowDepth = 2
+				};
+			}
+			else if(Control.Effect is DropShadowEffect)
+			{
+				Control.Effect = null;
+			} 
+
+			// Verify that the background is correctly set when the shadow is changed
+			UpdateBackground();
+		}
+
+		protected override UpdateBackground(){ 
+			// Enforce that a background color is set when the shadow is enabled
+			// to ensure, that the shadow is visible
+			if(Element.HasShadow && Element.BackgroundColor == Color.Transparent)
```

Comment:
So, this forces a white background when the background is set to transparent but a shadow is added as well? What is the alternative? It just doesn't show?

Reply:
Yes

Reply:
Alright, I guess both options are somewhat horrible. On the one hand people might start reporting that shadows aren't working, on the other hand that they get stuck with a white background.

I guess the latter is the better option in this case. At least they will see that the shadow is working, wonder why that is, go searching and find this code/comment or they will just set the color they want themselves.

I can live with this.

Reply:
> go searching and find this code/comment
:D

I thought the same that just not seeing the frame would be more irritating. When you see the white background you can guess at least, that you can override the color.